### ISSUE TITLE
A11y_AzureNotebooks_Jupyter_EnterExitRiseSlideshow_Keyboard: EnterExitRiseSlideshow Page is not accessible from keyboard using tab key.

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [jupyter/notebook#3976]
**EnvironmentDetails:**
Application Name:AzureNotebooks_Jyupter
Login Url: notebooks.azure.com
BrowserDetails:
Edge:Chrome:Version 68.0.3440.106 
Operating System:Windows10
 
 **Steps to reproduce:**
1.Login with the appropriate credentials to azure notebook account.    
2.Go to Get started à libraries à Select any of the libraries à click on clone to clone the library
3.Click on Run or right click and select Run to run the notebook.
5.Navigate to the EnterExitRiseSlideshow using the "Tab" key.
 
**Actual:**
**Issue1:**
EnterExitRiseSlideshow Page is not accessible from keyboard using tab key.
**Issue2:**
"?","Close" and the "Reveal Shortcuts Help" pop-up are not accessible using keyboard.

**Expected:**
**Issue1:**
EnterExitRiseSlideshow Page should be accessible from keyboard using tab key.
**Issue2:**
"?","Close" and the "Reveal Shortcuts Help" pop-up should be accessible using keyboard shorcuts  .

**UserImpact:**
User will face difficulty while traversing through the page. 

**Attachment:**
(https://github.com/jupyter/notebook/files/2400226/A11y_AzureNotebooks_Jupyter_EnterExitRiseSlideshow_Keyboard1.pptx)
(https://github.com/jupyter/notebook/files/2400234/A11y_AzureNotebooks_Jupyter_EnterExitRiseSlideshow_Keyboard2.pptx)

**Comment by jtpio** [comment]
Closing as there are now different issues tracking accessibility improvements for Jupyter Notebook 7, for example https://github.com/jupyter/notebook/issues/6800

Thanks!
